_includes/_cookie_banner.htm

```diff
@@ -0,0 +1,19 @@
+<div id="cookies-eu-banner" style="display: none;">
+    Do you accept the use of cookies by Google Analytics for statistical purposes?
```

Comment:
I think it would be good to note that necessary cookies relating to cloudflare are also used.

Reply:
Maybe smth like 
> Our CDN provider (Cloudflare) also uses cookies as part of their DDOS mitigation. Unfortunately we are not able to disable this cookie on our end.

Reply:
Yeah, I think that makes sense. Some of what I read recommended putting together a separate webpage with information about the cookies. Since we only have 2, and only one of them is optional, I thought that might be overkill, but I would be willing to put something together if that truly is best practice. Then we could keep this message succinct and link to a "Learn More" page

Reply:
In some examples i'm seeing the decline button states `Only use necessary cookies` instead.

Reply:
> Maybe smth like
> >    Our CDN provider (Cloudflare) also uses cookies as part of their DDOS mitigation. Unfortunately we are not able to disable these cookies on our end.

That wording seems very verbose to put on a banner. I'm not sure it's super relevant to put there anyway. The suggestion to change to "Only Use Necessary Cookies" may not be a bad idea though

Reply:
Or what about 
> Cloudflare also uses cookies. These can't be disabled.

 @RoboticMind?

Reply:
> That wording seems very verbose to put on a banner. I'm not sure it's super relevant to put there anyway. The suggestion to change to "Only Use Necessary Cookies" may not be a bad idea though

I believe it is in our best interests to be fully transparent about cookie use.

However in my opinion the best practice of creating a page that details what cookies are used is even more unnecessary, considering that only 3 cookies are used on the site (Ganalytics, Cloudflare, Cookie Banner)

Reply:
```suggestion
    Cloudflare uses cookies. These cookies can't be disabled.
    <br>
    Do you accept the use of cookies by Google Analytics for statistical purposes?
    <br>
```

Reply:
I like that. If I can wordsmith a little, what do you think of this?

```html
Cloudflare uses cookies that cannot be disabled.
<br>
Do you accept the use of cookies by Google Analytics for statistical purposes?
<br>

<button>Yes</button>
<button>Only use necessary cookies</button>
```

Reply:
sure @kjreills